caddy/setup/locale.go

```diff
@@ -0,0 +1,79 @@
+package setup
+
+import (
+	"strings"
+
+	"github.com/mholt/caddy/middleware"
+	"github.com/mholt/caddy/middleware/locale"
+	"github.com/mholt/caddy/middleware/locale/method"
+)
+
+// Locale configures a new Locale middleware instance.
+func Locale(c *Controller) (middleware.Middleware, error) {
+	locale := &locale.Locale{}
+
+	locales, methods, settings, err := localeParse(c)
+	if err != nil {
+		return nil, err
+	}
+
+	return func(next middleware.Handler) middleware.Handler {
+		locale.Next = next
+		locale.Locales = locales
+		locale.Methods = methods
```

Comment:
The naming is sparse. Go with »locale.FoundLocales«? »locale.ApplicableMethods«?